docs/metrics.md

```diff
@@ -0,0 +1,171 @@
+# METRICS
+
+## Data Analysis
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do users install and run this?
+
+What is the overall engagement of the Containers experiment?
+**This is the standard Daily Active User (DAU) and Monthly Active User (MAU) analysis.**
+
+This captures data from the users who have the add-on installed, regardless of
+whether they are actively interacting with it.
+
+![](images/kpi-1.png)
+
+### Immediate Questions
+
+* Do people use the containers feature & how do people create new container tabs?
+  * Click to create new container tab
+    * \+ `entry-point` value: "tab-bar" or "pop-up"
+* Do people who use the containers feature continue to use it?
+  * Retention: opening a second container tab (second tab in the same container, or a tab in a second container?)
+* What containers do people use?
+  * userContextId
+    * \+ Number of tabs in the container (when should we measure this? on every tab open?)
+* Do people edit their containers?
+  * Click on "Edit Containers"
+  * Click to edit a single container
+    * Click "OK"
+  * Click to delete a single container
+    * Click "OK"
+  * Click to add a container
+    * Click "OK"
+* Do people sort the tabs?
+  * Click sort
+    * \+ Number of tabs when clicked
+  * Average number of container tabs when sort was clicked
+* Do users show and hide container tabs?
+  * Click hide
+    * \+ Number of tabs when clicked
+    * \+ Number of hidden containers when clicked
+  * Click show
+    * \+ Number of tabs when clicked
+    * \+ Number of shown containers when clicked
+* Do users move container tabs to new windows?
```

Comment:
It will tell us if we should spend effort to implement "Move Containers to a new window" at the platform level.